mopro-ios/MoproKit/Example/MoproKit/AnonAadhaarViewController.swift

```diff
@@ -99,8 +103,12 @@ class AnonAadhaarViewController: UIViewController, WKScriptMessageHandler, WKNav
             // Record start time
             let start = CFAbsoluteTimeGetCurrent()
             
+            // Ark zkey url
+            let documentsUrl = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first!
```

Comment:
Is there some specific reason we are downloading via Swift as opposed to Rust? Seems like we'd have to implement same logic in many places, is it that much easier?

Reply:
Interesting, no idea why!

Yes, that's a fair point.

Not sure which is best. If you got this to work let's start with this and in the future we can re-evaluate if needed!